README.md

```diff
@@ -204,22 +212,29 @@ To deploy the EVM rollup anchor, follow these steps:
 
 Find a reference script [here](./evm/scripts/deploy-test.ts).
 
-The Substrate pallet and ink! anchor deployment docs are currently under development (TODO).
+To deploy the Ink rollup anchor, follow these steps:
+
+1. Deploy the Phat Contract with a pre-generated ECDSA key pair (called submission key)
```

Comment:
I was inspired by the documentation for EVM side and when I read it again I have the feeling that the reader can understand that the attestor key is given in paramater of the constructor. But it's incorrect. So I reformulated the sentences for the ink! part.

Reply:
It was once called "submission key", but since we have added the MetaTx support, it's changed to "attestor key". Here the comment just refers to the naming of the key.

Reply:
Got it. The documentation has been updated.